src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Liberation Sans,Arial,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
Dan,
it took a while to submit you another pull-request. I gave a typography class 10 years ago and I took the time to update my knowledge. Concerning the default font choice I suggest the following based on this:

1. Web-fonts and print-fonts should not be mixed. They have different design goals.
2. I gave priority have open license fonts. If they are not installed Microsoft proprietary core fonts are chosen.
3. I based my choice on DejaVu and Liberations fonts. They are [available on all systems with LibreOffice installed](http://superuser.com/questions/611583/which-fonts-come-with-openoffice-libreoffice). *) If not already installed by default all Linux distribution have packages ready for these fonts.
4. DejaVu is used in all shapes, except for pick-font mono, there I suggest Liberation mono. It has this "console"-feeling we want and harmonizes well.
5. I left the hyphenate switch on. By default no hyphenation will occur as long as the user does not install additional hyphenation patterns. But when he does he does not need to search for the switch to enable hyphenation.

Best 


Jens
PS: *)  Last not least fonts are matter of taste: here you find [other common Linux fonts](https://mondaybynoon.com/linux-font-equivalents-to-popular-web-typefaces/). I did not find any recent statistics but according to [this source](http://www.ehow.com/info_8203711_7-common-fonts.html) _Century Schoolbook L_ is one of the most common Linux fonts, with a 98.03 percent installation rate. It is very traditional and high quality font that supports many many scripts.